standards/WIT-SVID.md

```diff
@@ -1,4 +1,321 @@
+
+## Table of Contents
+
+## 1. Introduction
+
+The Workload Identity Token (WIT) is a token format specified by the IETF WIMSE working group in the [WIMSE Workload Credentials][1] document. The WIT binds a public key to the identity of a workload.
+
+The WIT-SVID is a sub-profiling of this token format for use in SPIFFE contexts. This document does not redefine key elements specified within the upstream [WIMSE Workload Credentials][1] document and as such the reader should be familiar with this document and its contents before implementing WIT-SVID.
+
+WIT-SVIDs are [JSON Web Tokens (JWT)][6] encoded using [JSON Web Signature (JWS)][2] compact serialization.
+
+### 2. JOSE Header
+
+This section describes the JOSE header parameters that are defined for the WIT-SVID.
+
+The WIT-SVID specification does not introduce any JOSE header parameters beyond those defined by the upstream document for the WIT. However, it does set additional restrictions and provide SPIFFE-specific guidance on some parameters.
+
+### 2.1. Key ID - `kid`
+
+Unique identifier of the key-pair used by the issuer to sign the WIT-SVID. The `kid` header parameter is defined by the [JSON Web Signature (JWS)][2] document.
+
+For a WIT-SVID, this parameter MUST be present. This differs from the upstream WIT itself and the JWT-SVID where this parameter is optional.
+
+The format of the value of this parameter is unspecified and it MUST be treated by verifiers as a case-sensitive string.
+
+The issuer MUST ensure that the value set within the `kid` parameter is unique to each issuing key-pair.
+
+This parameter allows a validator to select the correct public key from a trust bundle for validating the signature of a WIT-SVID without enumerating all keys.
+
+### 2.2. Type - `typ`
+
+The `typ` header parameter is defined by the [JSON Web Signature (JWS)][2] document.
+
+For a WIT-SVID, this parameter MUST be present and MUST be set to `wit+jwt`.
+
+### 2.3. Algorithm - `alg`
+
+Identifies the cryptographic algorithm used to sign the WIT-SVID. The `alg` header parameter is defined by the [JSON Web Signature (JWS)][2] document.
+
+For a WIT-SVID, this parameter MUST be present and set to one of the following supported values:
+
+`alg` Param Value | Digital Signature Algorithm
+------------------|-----------------------------
+RS256 | RSASSA-PKCS1-v1_5 using SHA-256
+RS384 | RSASSA-PKCS1-v1_5 using SHA-384
+RS512 | RSASSA-PKCS1-v1_5 using SHA-512
+ES256 | ECDSA using P-256 and SHA-256
+ES384 | ECDSA using P-384 and SHA-384
+ES512 | ECDSA using P-521 and SHA-512
+PS256 | RSASSA-PSS using SHA-256 and MGF1 with SHA-256
+PS384 | RSASSA-PSS using SHA-384 and MGF1 with SHA-384
+PS512 | RSASSA-PSS using SHA-512 and MGF1 with SHA-512
+
+Validators MUST reject WIT-SVIDs with an unsupported `alg` parameter value.
+
+### 2.3. Additional Header Parameters
```

Comment:
From SIG-SPEC: Rough consensus that we should acknowledge the existence of crit - and tell validators to follow the rules set out by RFC7515.